### PR TITLE
#3236 enable ohara to deploy multiples jars in a stream app

### PR DESCRIPTION
**Pull request by chia7712**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #3236)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by jackyoh** [comment]
很像有幾支測試程式資料型態有問題

**Comment by chia7712** [comment]
> 很像有幾支測試程式資料型態有問題

not really. the root cause is the loading definitions from file is not working.

**Comment by jackyoh** [comment]
> > 很像有幾支測試程式資料型態有問題
> 
> not really. the root cause is the loading definitions from file is not working.

了解，Thank you.

**Comment by jackyoh** [comment]
retry client

**Comment by chia7712** [comment]
@jackyoh thanks for the retry but I don't have time to fix it :(

**Comment by chia7712** [comment]
retry k8sConfigurator

**Comment by chia7712** [comment]
@oharastream/frontend could you give me some hint of APIs error?

**Comment by chia7712** [comment]
> could you give me some hint of APIs error?

oh. I fix it already :)

**Comment by chia7712** [comment]
retry sshConnector

**Comment by chia7712** [comment]
retry k8sConnector

File: ohara-configurator/src/main/scala/com/island/ohara/configurator/route/InspectRoute.scala [hunk 1]
```diff
@@ -187,34 +189,24 @@ private[configurator] object InspectRoute {
...
-                    sinks.map(n =>
-                      ClassInfo(classType = SINK_CONNECTOR_KEY, className = n, settingDefinitions = Seq.empty)) ++
-                    streamApps.map(n =>
-                      ClassInfo(classType = STREAM_APP_KEY, className = n, settingDefinitions = streamDefinitions))
-                )
-            })
+        complete(dataStore.value[FileInfo](ObjectKey.of(group, fileName)).flatMap { fileInfo =>
+          val (sources, sinks, streamApps) = ReflectionUtils.loadConnectorAndStreamClasses(fileInfo)
+          val sourcesInfos = sources.map(n => ClassInfo.sourceConnector(className = n, settingDefinitions = Seq.empty))
+          val sinkInfos = sources.map(n => ClassInfo.sinkConnector(className = n, settingDefinitions = Seq.empty))
```
Comment (saivirtue) [inline ohara-configurator/src/main/scala/com/island/ohara/configurator/route/InspectRoute.scala]: source => sink ?

Comment (chia7712) [reply ohara-configurator/src/main/scala/com/island/ohara/configurator/route/InspectRoute.scala]: nice finding

**Comment by chia7712** [comment]
retry manager-api

**Pull request closed without merging**